.env

```diff
@@ -1,7 +1,7 @@
 COMPOSE_PROJECT_NAME=ereolen
 COMPOSE_DOMAIN=ereolen.local.itkdev.dk
-REMOTE_HOST=ereolen-p02.dbc.dk
-REMOTE_DB_DUMP_CMD='drush --root=/data/www/ereolen_dk/htdocs --uri=ereolen.dk sql-dump'
-REMOTE_PATH='/data/www/ereolen_dk/htdocs/sites/default/files'
+REMOTE_HOST=aarhus-s02.dbc.dk
+REMOTE_DB_DUMP_CMD='drush --root=/data/www/stg_ereolen_dk/htdocs --uri=stg.ereolen.dk sql-dump'
+REMOTE_PATH='/data/www/stg_ereolen_dk/htdocs/sites/default/files'
```

Comment:
I'm not sure we should change this – or commit it in the first place, but that's another issue.

Reply:
Well the dbc reference isn't right as that is no longer the prod server.